.idea/codeStyles/Project.xml

```diff
@@ -32,16 +32,7 @@
     <JetCodeStyleSettings>
       <option name="PACKAGES_TO_USE_STAR_IMPORTS">
         <value>
-          <package name="kotlinx.android.synthetic" alias="false" withSubpackages="true" />
```

Comment:
Hi @mzorz do we want to include these changes here? They are related to how different versions of Android Studio manipulates this file right?

Reply:
you're right! I think it's handled in a later PR as well but we can definitely remove these changes from this PR in particular if it's creating noise

Reply:
Okay if it handled in the later one then it's fine since we are merging these changes into the feature branch 👍